app/database.py

```diff
@@ -0,0 +1,9 @@
+import sys
+import tinydb
+
+from pathlib import Path
+
+# DB initializtion
+DBJSON_PATH = Path(sys.path[1]) / Path("db.json")
+print(DBJSON_PATH)
```

Comment:
this seems to be used for debugging, can you remove it if it is not needed anymore

Reply:
also why are we putting the database in the sys.path?

Reply:
Yup, I'll delete the print statement
Actually it'll be updated in the next commit